caproto/_broadcaster.py

```diff
@@ -84,11 +96,11 @@ def send(self, *commands):
         total_commands = len(commands)
         tags = {'role': repr(self.our_role)}
         for i, command in enumerate(commands):
-            if hasattr(command, 'name'):
-                tags['pv'] = command.name
-            for address in self._our_addresses:
-                tags['our_address'] = address
-                search_logger.debug(f"%d of %d %r", 1 + i, total_commands, command, extra=tags)
+            tags['counter'] = (1 + i, total_commands)
+            if isinstance(command, (SearchRequest, SearchResponse)):
+                self.search_log.debug("%r", command, extra=tags)
+            else:
+                self.log.debug("%r", command, extra=tags)
```

Comment:
we are sure that the logging code does not hold on a reference to this dict?

Reply:
Yes.